src/main/java/com/autotune/database/helper/DBHelpers.java

```diff
@@ -149,7 +149,7 @@ public static void setRecommendationsToKruizeObject(List<ListRecommendationsAPIO
             }
 
             // Check if Container data map and namespace data both are not empty
-            if (containerDataMap.isEmpty() && null == k8sObject.getNamespaceData()) {
+            if (containerDataMap.isEmpty() && null == k8sObject.getNamespaceDataMap()) {
```

Comment:
What if `getNamespaceDataMap()` is empty ?

Reply:
added the empty check